public/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "short_name": "diaspora*",
+  "name": "diaspora*",
```

Comment:
The old "Add to Home" used the page title, which used [`pod_title`](https://github.com/diaspora/diaspora/blob/9bab794/config/diaspora.yml.example#L356-L358) from the `diaspora.yml`. Should we use a controller to generate the json so we can also use the dynamic title?